.github/actions/report-job-status/README.md

```diff
@@ -0,0 +1,31 @@
+# Send VK Teams notification on failure
+
+This action uses [report-job-status action](https://github.com/tarantool/actions/tree/master/report-job-status)
+and chooses the proper chat for the notification on failure:
+* The team chat (Tarantool CI/CD reports) - if the job started on `master` and
```

Comment:
... - if the job started on any event in the `master` or release branch (`1.10`, `2.10`, etc)

Reply:
done